ambry-clustermap/src/main/java/com.github.ambry.clustermap/HelixClusterAggregator.java

```diff
@@ -62,10 +62,12 @@
         combine(partitionSnapshot, snapshotWrapperCopy, statsWrapperJSON.getKey(), partitionTimestampMap);
       }
     }
-    logger.info("Reduced raw snapshot {}", mapper.writeValueAsString(rawPartitionSnapshot));
-    logger.info("Reduced snapshot {}", mapper.writeValueAsString(partitionSnapshot));
+    LOGGER.trace("Combined raw snapshot {}", mapper.writeValueAsString(rawPartitionSnapshot));
```

Comment:
Can you surround this with an `if (logger.isTraceEnabled())` since it requires deserializing the full snapshot.